src/Rule/Callback.php

```diff
@@ -79,11 +79,23 @@ public function afterInitAttribute(DataSetInterface $dataSet): void
             return;
         }
 
-        try {
-            $this->callback = Closure::fromCallable([$dataSet->getObject()::class, $this->method]);
-        } catch (TypeError) {
-            throw new InvalidArgumentException('Method must exist and have public and static modifiers.');
+        if ($this->method === null) {
+            return;
```

Comment:
A test is needed for a case when no method was passed.

Reply:
It is already present https://github.com/yiisoft/validator/blob/72cf6616a62d3faa87c1b6897741a9d7e4c97d59/tests/Rule/CallbackTest.php#L61

Reply:
Actually, let's leave it for later. There are other non-covered parts in Callback rule (not related with this PR's diff) and we have a task of raising code coverage close to 100% before 1.0.0 release anyway.